### PR TITLE
Bump live-common

### PR DESCRIPTION
**Pull request by gre**
like https://github.com/LedgerHQ/ledger-live-desktop/pull/1936 but on mobile

### CODE REVIEW

File: ios/ledgerlivemobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1371,6 +1364,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```
Comment (valpinkman) [inline ios/ledgerlivemobile.xcodeproj/project.pbxproj]: same changes that I had O_O

Comment (gre) [reply ios/ledgerlivemobile.xcodeproj/project.pbxproj]: I guess it's ok

**Comment by gre** [comment]
@Arnaud97234 these are invalid path and was intentionally removed from scanning on mobile as we chat about it. If users need to recover invalid path funds, they will need the Desktop app.

i'm fixing the libcore issue on the other hand.

**Comment by Arnaud97234** [comment]
> @Arnaud97234 these are invalid path and was intentionally removed from scanning on mobile as we chat about it. If users need to recover invalid path funds, they will need the Desktop app.
> 
> i'm fixing the libcore issue on the other hand.

**Pull request merged**